pkg/storage/unified/resource/resource.proto

```diff
@@ -448,6 +448,118 @@ message HealthCheckResponse {
   ServingStatus status = 1;
 }
 
+// ResourceTable is a protobuf variation of the kubernetes Table object.
+// This format allows specifying a flexible set of columns related to a given resource
+message ResourceTable {
+  // Columns describes each column in the returned items array. The number of cells per row
+  // will always match the number of column definitions.
+  repeated ResourceTableColumnDefinition columns = 1;
+
+  // rows is the list of items in the table.
+  repeated ResourceTableRow rows = 2;
+
+  // When more results exist, pass this in the next request
+  string next_page_token = 3;
```

Comment:
do we need these last 3 fields? Looks like they're already part of `ListMeta` in the table.

Reply:
nvm!